bmi-guidance/README.md

```diff
@@ -0,0 +1,43 @@
+# BMI Health Check
+
+This project shows how you can easily train machine learning models from human input to extend human capabilities and reduce the workload of experts handling a high rate of incoming requests.
+
+### Use case
+In this demo, we implemented a health assistant that helps you identify potential obesity issues in children and adolescents. 
+Apart from being a real-life problem, it's interesting because we modelled two different personas whose relationship we're trying to mediate: the patients sending the requests, and the doctor answering them.
+
+
+We approach this problem by introducing two types of agents handling the specific needs of these two respective types of users: the Patient Agent makes sure the patient checks their child's weight regularly, and gets expert opinion on it as soon as possible after reporting it.
+The Doctor Agent communicates the incoming patients' requests to the doctor, but also wants to control his/her workload and make sure he/she is not overloaded with work. That's why we add an extra machine learning component that gradually learns from the doctor's responses.
+
+
+With time, the model itself will take over most of the workload, leaving only the "trickier" edge cases for the doctor to handle. 
+Ideally, this is how we want to use ML models in general: not to substitute humans, but to extend their capabilities by automating the straightfowrard, repetitive work, and allowing them to focus more on the creative, more demanding tasks.
+
+
+### Running the demo
+To run, first set your [TELEGRAM_BOT_TOKEN](https://core.telegram.org/bots) and [OPENAI_API_KEY](https://beta.openai.com/account/api-keys) in `.env`. Additionally, you will need to set the credentials for accessing storage on Azure:
+```
+CLOUD_TYPE=AZURE
+AZURE_ACCOUNT_URL="https://bmiguidance.blob.core.windows.net"
+AZURE_ACCOUNT_KEY="3b4PVRaRisL6JcLeMbFlJDddyWP61/lqNFeTuPORfHwZNZS/cawTyfg+gWldH9hmruLRRxcgWcys+AStzBHOfQ=="
+AZURE_MEDIA_CONTAINER="media"
```

Comment:
Let's fix Telegram Adapter so that we can send public links and then we don't need this.